apps/ios/Shared/Views/NewChat/NewChatView.swift

```diff
@@ -693,6 +693,7 @@ struct ScannerInView: View {
                             .frame(maxWidth: .infinity, maxHeight: .infinity)
                             .foregroundColor(Color.clear)
                         switch cameraAuthorizationStatus {
+                        case .authorized, nil: EmptyView()
```

Comment:
Avoids flashing "Tap to scan" button, if user has already granted camera permission.